logrotate.c

```diff
@@ -607,16 +607,16 @@ static int compressLogFile(char *name, struct logInfo *log, struct stat *sb)
     }
 
     close(compressPipe[1]);
-	while ((i = read(compressPipe[0], buff, sizeof(buff) - 1)) > 0) {
-		if (!error_printed) {
-			error_printed = 1;
-			message(MESS_ERROR, "Compressing program wrote following message "
-					"to stderr when compressing log %s:\n", name);
-		}
-		buff[i] = '\0';
-		fprintf(stderr, "%s", buff);
-	}
-	close(compressPipe[0]);
+    while ((i = read(compressPipe[0], buff, sizeof(buff) - 1)) > 0) {
+            if (!error_printed) {
+                error_printed = 1;
+		message(MESS_ERROR, "Compressing program wrote following message "
+                               "to stderr when compressing log %s:\n", name);
+            }
+            buff[i] = '\0';
+            fprintf(stderr, "%s", buff);
+    }
+    close(compressPipe[0]);
```

Comment:
This hunk actually breaks the indentation (random tabs replaced by spaces).  I will tweak the commit not to do so.